DCP_Tutorial.md

```diff
@@ -0,0 +1,135 @@
+2) After the submission moves into the `Completed` state, you should now see your bundle in the `Submit` tab! Go ahead and click on the `dss api` link to see the bundle manifest stored in the DSS. Each bundle in the data store has a fully qualified ID (FQID) that is composed of a UUID and Version. The bundle manifest is generated by Ingest and is a composition of the file FQIDS mentioned above. Your bundle and files were stored in AWS and are now being synced over to GCP. You will notice that some of the files in the manifest are indexed and others are not. In general, all metadata files are indexed in order to allow for searchability via the data store's elastic search cluster.
+3) Lets go ahead and download the bundle using the HCA cli. This step will download the bundle into a directory named after bundle UUID.
+        hca dss download --bundle-uuid {BUNDLE_UUID_FROM_STEP_2) --replica {AWS OR GCP}
+4) Go ahead and open up some of the files in the directory. You should notice that these metadata files have values tracing all the way back to the spreadsheet. The structure of the metadata files is defined by the schema as found at https://github.com/HumanCellAtlas/metadata-schema.
+
+#### Phase 3: Analysis
+**Components:** Analysis, Data Store, Ingest, Upload  
+**Description:** In this phase, you will be taking a look at the analysis workflow corresponding with your bundle. You will follow the workflow as it completes and deposits the analysis result back into the data store via Ingest/Upload. Analysis is currently running workflows via cromwell in GCP infrastructure.
+1) Go back to your submission in the Ingest UI and copy the `Project UUID` from the `Overview` tab.
+2) Now replace the project uuid in the URL and visit the Job Manager UI at https://job-manager.caas-prod.broadinstitute.org/jobs?q=project_uuid%3D{PROJECT_UUID_FROM_ABOVE_STEP}. You should now see a workflow running corresponding with your bundle.  The data pipelines component maintains a subscription to the data store in order to trigger workflows. When a bundle is synced in GCP that meets the query, the data store notifies the analysis component. The rules of the subcription are based on metadata fields that are currently supported by production pipelines (factors like species, library construction approaches (10x/ss2), etc). The current subscriptions can be found at https://github.com/HumanCellAtlas/lira/tree/master/subscription/elasticsearch_queries.
```

Comment:
Note: you won't be able to see anything on https://job-manager.caas-prod.broadinstitute.org/jobs?q=project_uuid%3D{PROJECT_UUID_FROM_ABOVE_STEP} unless you have registered on Terra and send your user email to analysis to be whitelisted. Currently, the process is documented briefly here: https://github.com/HumanCellAtlas/secondary-analysis/issues/237 But we will create a better doc for that later soon.

Reply:
Sounds good. I have a little section in the Requirements doc on how to request access. We can link your doc once its up

Reply:
@rexwangcc in the meantime, I've pointed people to ask for access in #dcp-ops-help

Reply:
I've updated instructions to point to the ticket to get access to the job manager